db/Dockerfile

```diff
@@ -1,16 +1,15 @@
-FROM postgres:9.4
+FROM postgres:9.5
```

Comment:
This might lead to error according to the commit message here: https://github.com/mattermost/mattermost-docker/commit/57d1d448b2a3aaea930a552d10e9ba6bd6888df3.